README.md

```diff
@@ -79,52 +83,70 @@ substoich =[2 0 0;
             0 1 0;
             1 1 0;
             0 0 1;
-            0 0 3]'
+            0 0 3]
 prodstoich = [0 1 0;
               2 0 0;
               0 0 1;
               1 1 0;
-              3 0 0]'
-prn = loadrxnetwork(MatrixNetwork(), pars, substoich, prodstoich; 
-                    species=species, params=pars)
+              3 0 0]
+mn= MatrixNetwork(pars,substoich,prodstoich;params=pars) # matrix network
+prn = loadrxnetwork(mn) # dense version
+# test the two networks are the same
+@assert rs == prn.rn
+
+stoichmat =[2  0  1  0  0  3;
+                 0  1  1  0  0  0;
+                 0  0  0  1  3  0]
+incidencemat  = [-1   1   0   0   0;
+                 1  -1   0   0   0;
+                 0   0  -1   1   0;
+                 0   0   1  -1   0;
+                 0   0   0   0  -1;
+                 0   0   0   0   1]
+cmn= ComplexMatrixNetwork(pars,stoichmat,incidencemat;params=pars)  # complex matrix network
+prn = loadrxnetwork(cmn)
 
 # test the two networks are the same
 @assert rs == prn.rn
 ```
 
-The basic usage is
+The basic usages are
 ```julia
-prn = loadrxnetwork(MatrixNetwork(),  
-                    rateexprs::AbstractVector, 
-                    substoich::AbstractMatrix, 
-                    prodstoich::AbstractMatrix; 
-                    species::AbstractVector=Operation[], 
-                    params::AbstractVector=Operation[])
+prn = loadrxnetwork(mn::MatrixNetwork) #or
+prn = loadrxnetwork(cmn::ComplexMatrixNetwork)
 ```
-Here `MatrixNetwork()` is the dispatch type, which selects that we are
-constructing a matrix-based stoichiometric representation as input. The other
-parameters are:
+Here `MatrixNetwork` and `ComplexMatrixNetwork` are the types, which selects that we are
+constructing a substrate/products stoichiometric matrix-based or complex- matrix based stoichiometric representation as input. These types have 
+following fields :
 - `rateexprs` - Any valid `ModelingToolkit.Operation`s for the rates, or basic
   number types. This can be a hardcoded rate constant like `1.0`, a a parameter
   like `k1` above, or an `Operation` involving parameters and species like
   `k*A`. Note, the reaction `A+B --> C` with rate `k*B` would have rate law
   `k*A*B^2`.
-- `substoich` - A number of species by number of reactions matrix with entry
-  `(i,j)` giving the stoichiometric coefficient of species `i` as a substrate in
-  reaction `j`. 
-- `prodstoich` - A number of species by number of reactions matrix with entry
-  `(i,j)` giving the stoichiometric coefficient of species `i` as a product in
-  reaction `j`.
+- matrix inputs 
+  - For `MatrixNetwork`
+    - `substoich` - A number of species by number of reactions matrix with entry
```

Comment:
If you drop the transpose above than it is no longer number of species by number of reactions as the docs say, so there should be a consistency in the docs and input.

Reply:
> Actually, I think we just need to fix Catalyst to make the stoichiometry matrices species by reactions since this is more standard (at least it is what Wikipedia says).
> 
> I'm making some edits / changes to your Catalyst PR that will flip the matrices to be species by reactions. Once we have a breaking release there you can update the PR here to be consistent with Catalyst.

That makes sense.. I think numspecies x numreactions is sounds standard from 
`du/dt= S*f(u,p,t)` perspective too. Currently to calculate `S` i need to take it equal to `transpose(netstoichmat(network))`